src/browser/BrowserAction.cpp

```diff
@@ -240,14 +240,16 @@ QJsonObject BrowserAction::handleGetLogins(const QJsonObject& json, const QStrin
     entryParameters.dbid = id;
     entryParameters.hash = browserRequest.hash;
     entryParameters.siteUrl = siteUrl;
-    entryParameters.formUrl = formUrl;
```

Comment:
Did you intend to remove this line?

Reply:
Nope. For some reason it was in the original PR. Setting it back.